VropsCollector.py

```diff
@@ -51,37 +51,16 @@ def __init__(self, target):
         self._target = target
         self._user = os.environ['USER']
         self._password = os.environ['PASSWORD']
+        modules = tools.get_modules.get_modules()
+        self._modules = modules[1]
+        self._modules_dict = dict()
+        for module in self._modules:
+            self._modules_dict[module] = importlib.import_module(module, modules[0])
 
     def collect(self):
-        if os.environ['DEBUG'] == '1':
-            print('started')
+        for target in self._modules_dict.keys():
+            mod = self._modules_dict[target]
+            func = getattr(mod, target)
+            res = func(self._target, self._user, self._password).collect()
+            yield res
```

Comment:
if you are directly yielding the result, you will only be able to have a single metric per module.
Iterate over the results and yield every item.